tests/EndToEnd/Bugs/errors.twig

```diff
@@ -0,0 +1,7 @@
+{% if foo is defined %}{{ foo }}{% endif %}
+
+{% set languagesTo = ''|split(',')|filter(v => v != null) %}
```

Comment:
The closure/arrow function seems to have issues

Reply:
Created a PR with the failing test here #36 

Not yet solved